resources/views/admin/multiegg/support.blade.php

```diff
@@ -52,7 +52,7 @@
 		    <textarea name="body" class="form-control no-resize" rows="5"></textarea>
 		</div>
 		<div>
-		    <textarea class="form-control no-resize" rows="2" readonly>By submitting this form, you consent for your panel Domain, IP, Addon Version, and other addons / license details to be shared with our support team. (We will get back to you within 3 business days)</textarea>
+		    <textarea class="form-control no-resize" rows="2" readonly>By submitting this form, you consent for your panel Domain, IP, Addon Version, and other addons / license details to be shared with our support team. (We will get back to you within 1-3 business days)</textarea>

```

Comment:
Don't exactly see the need to put a `1-3` here instead of just a `3`

We will get back to you within 1-3 business days
We will get back to you within 3 business days

Within generally implies the 1 is already there, as within, is well, less than or equal to.